lib/recaptcha.rb

```diff
@@ -50,7 +51,7 @@ def self.get(verify_hash, options)
     else
       Net::HTTP
     end
-    query = URI.encode_www_form(verify_hash)
+    query = Rack::Utils.build_query(verify_hash)
```

Comment:
Thinking about just adding a simple method for this, instead of a new dependency, but `rack` is used all over the place, so didn't seem like such a bad one to bring in.